src/Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -97,10 +97,15 @@ public NetworkManager(IConfigurationManager configurationManager, IConfiguration
         _networkEventLock = new object();
         _remoteAddressFilter = new List<IPNetwork>();
 
+        var detectNetworkChange = startupConfig[DetectNetworkChangeKey] == bool.TrueString;
```

Comment:
Should use `bool.TryParse` here (or at least use case-insensitive since the value is "True"`)